### DISCUSSION TITLE
Feature Request: Firefox/cross-browser version.

### DISCUSSION THREAD

**Discussion by simonwiles** [coddingtonbear/obsidian-web#46] [Ideas]
I just came here from #5, and noticed that this feature request never made it over to the discussions section, so I'm starting a discussion here now.

@coddingtonbear do you have any sense of how much might be involved in building a cross-browser extension from this codebase (and/or how manifest v3 might alter this calculus)?  I made a few Firefox extensions back in the pre-WebExtension APIs days, but I've not stayed current with this stuff.  I'm sufficiently interested in a Firefox version of this extension that I might try to find some time to work on it.

**Comment by adityaprasoon** [comment]
A firefox extension is very much needed for those last people still clenching to firefox. @simonwiles I will be interested to contribute if community work starts on it.

**Comment by coddingtonbear** [comment]
Although this is technically the first conversation on this, the most discussion has occurred in #79, so I'm going to close this as a duplicate of that.
